src/index.js

```diff
@@ -1,4 +1,12 @@
-const roads = require('../src/config/roads.js');
-const buildVillageMap = require('./villagemap.js');
+const _ = require('lodash');
+const roads = require('../src/config/roads');
+const { buildVillageMap, toListStates } = require('./villagemap');
```

Comment:
in general function are named after actions: `toListStates` doesn't sound like an action, `listStates` or `getStates` sounds more like an action.
You can look for articles that talk about that, one of them is: https://dmitripavlutin.com/coding-like-shakespeare-practical-function-naming-conventions/